purchase_work_acceptance/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_work_acceptance,access_work_acceptance,model_work_acceptance,purchase.group_purchase_manager,1,1,1,1
```

Comment:
Do we need to be purchase manager to access this model?
I think only purchase user should be enough, WDYT?

Reply:
I agree. I will change it.